Setup/SetupAssist/Checks/UserContext/Test-UserGroupMemberOf.ps1

```diff
@@ -13,22 +13,12 @@ function Test-UserGroupMemberOf {
         [bool]$PrepareDomainOnly
     )
 
-    $whoami = whoami
-    $whoamiAllOutput = whoami /all
-    $userSid = ($whoamiAllOutput | Select-String $whoami.Replace("\", "\\")).Line.Replace($whoami, "").Trim()
-
-    $params = @{
-        TestName = "User Administrator"
-        Details  = "$whoami $userSid"
+    if ($null -eq $Script:UserSid) {
```

Comment:
I'd like to avoid the use of a script-scoped variable.

Also, I'm not clear why we're calling whoami /all to get the SID when `[Security.Principal.WindowsIdentity]::GetCurrent().User.Value` already has it. That's what Confirm-Administrator is using. Do we need to use whoami at all? Is it just for output?

Reply:
@bill-long check out latest changes